grassroot-integration/src/main/java/za/org/grassroot/integration/location/LocationInfoBrokerImpl.java

```diff
@@ -94,7 +113,7 @@ public void setJwtService(JwtService jwtService) {
 
     @PostConstruct
     public void init() {
-        placeLookupLambda = environment.getProperty("grassroot.places.lambda.url", "http://localhost:3000");
+        placeLookupLambda = environment.getProperty("grassroot.places.lambda.url", "https://l48oimxk54.execute-api.eu-west-1.amazonaws.com/latest");
```

Comment:
Don't do this. Reverse this before committing. Very important. Please do not do again.